tests/acceptance/support/sandbox.rs

```diff
@@ -460,6 +460,20 @@ impl SandboxBuilder {
         self.add_exec_dir_to_path()
     }
 
+    /// Add an arbitrary file to the test project within the sandbox,
+    /// give it executable permissions,
+    /// and add its directory to the *front* of PATH, shadowing any volta binaries.
```

Comment:
nit: Can we update this doc comment to more accurately reflect what the method does? It appears that it doesn't work with any files, rather it only does the prepending. Can we also document the constraint that `prepend_exec_dir_to_path` is exclusive of `add_exec_dir_to_path`?